README.md

```diff
@@ -72,10 +72,24 @@ make test
 
 The library files and dictionaries (`libedm4hep.so`, ...) must be put somewhere in `LD_LIBRARY_PATH`.
 
-
-
-
-
+## Python bindings
```

Comment:
Maybe clarify here that this is only necessary when users are not reading stuff via the podio python bindings. Because they will take care of everything for only working with collections read form file.

Reply:
Done